documentation/source/user_section/tutorials/lesion-analysis/lesion-metrics-template.rst

```diff
@@ -0,0 +1,30 @@
+Template/Atlas-based lesion analysis
+####################################
+
+The ``sct_analyze_lesion`` function allows you to provide a folder containing an atlas/template registered to an anatomical image.
+If specified, the function computes:
+
+* a. for each lesion, the proportion of that lesion within each vertebral level and each region of the template (e.g. GM, WM, WM tracts). Each cell in the output XLS file contains a percentage value representing how much of the lesion volume exists within the region indicated by the row/column (rows represent vertebral levels, columns represent ROIs). The percentage values are summed to totals in both the bottom row and the right column, and the sum of all cells is 100 (i.e. 100 percent of the lesion), found in the bottom-right.
+* b. the proportions of each ROI (e.g. vertebral level, GM, WM) occupied by lesions.
+
+.. note::
+
+   TODO: as we need to provide a template/atlas, which is covered by the next tutorial (see below), we can consider moving this tutorial after the template/atlas-based lesion segmentation tutorial.
+
+   You can register the template and warp the atlas to the anatomical image using the ``sct_register_to_template`` and ``sct_warp_template`` functions, respectively.
+   See :ref:`template-registration <template-registration>` for more information.
```

Comment:
I think we can remove this TODO? It seems straightforward to me to refer users to another tutorial to complete the necessary steps.

```suggestion
.. note::

   The following command requires that you register the PAM50 template to the ``t2_lesion.nii.gz`` anatomical image. You can register the template and warp the atlas to the anatomical image using the ``sct_register_to_template`` and ``sct_warp_template`` functions, respectively.
   See :ref:`template-registration <template-registration>` for more information.
```